.circleci/config.yml

```diff
@@ -155,7 +155,7 @@ jobs:
                     -kubecontext="kind-dc1" \
```

Comment:
Does it make sense to change references to lifecycle-sidecar in hack/helm-reference-gen to also use consul-sidecar? I figure the exact values are getting stale anyway and meant to test that nested nodes work, etc.

Reply:
Yup!! That sounds right. It would be nice if the fixture looked even lesser like our values file because it definitely is confusing. I don't think it needs to be updated and can be left as is.